pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java

```diff
@@ -2546,6 +2546,45 @@ protected void internalSetMaxTopicsPerNamespace(Integer maxTopicsPerNamespace) {
         internalSetPolicies("max_topics_per_namespace", maxTopicsPerNamespace);
    }
 
+   protected void internalSetProperty(String key, String value) {
+       validatePoliciesReadOnlyAccess();
+       try {
+           final String path = path(POLICIES, namespaceName.toString());
+           updatePolicies(path, (policies) -> {
+               policies.properties.put(key, value);
```

Comment:
Do we need to provide an option for overwriting the existing key?

Reply:
Overwriting the existing key is the default behavior. If we need to support prevent an update when the key exists, we can add it in the future.